gamemode/modules/base/sh_util.lua

```diff
@@ -105,24 +105,27 @@ function DarkRP.findPlayers(info)
         -- Playerinfo is always to be treated as UserID when it's a number
         -- otherwise people with numbers in their names could get confused with UserID's of other players
         if tonumber(PlayerInfo) then
-            if IsValid(Player(PlayerInfo)) and not found[Player(PlayerInfo)] then
-                found[Player(PlayerInfo)] = true
+            local foundPlayer = Player(PlayerInfo)
+            if IsValid(foundPlayer) and not found[foundPlayer] then
+                found[foundPlayer] = true
                 players = players or {}
-                table.insert(players, Player(PlayerInfo))
+                table.insert(players, foundPlayer)
             end
             continue
         end
 
+        local stringPlayerInfo = string.lower(tostring(PlayerInfo))
```

Comment:
This data was being converted to lowercase twice for every player in the loop below. If you have 128 players (with no found player), it would do this string conversion 128 times! Take it out of the loop, and it's done only once for every player.